blazar_dashboard/content/leases/templates/leases/_widget_computehost_capabilities.html

```diff
@@ -1,4 +1,4 @@
-<div id="criteria-computehost" style="margin-bottom:50px">
+<div id="criteria-computehost" style="margin-bottom:50px" switchable_class="{{ widget.switchable_class }}">
```

Comment:
When you're doing something like this, data attributes are the way to go. I don't think it _really_ matters that much, but this is atypical HTML, and in any event, if you need to attach things to the DOM that are not related to presentation, data attributes are designed for this.

```html
<div data-switchable-class="{{ widget.switchable_class }}">
  ...
</div>
```

You can access data attributes with the `.dataset` property, and you access the attributes by referencing them by name. The browser exposes them in camelCase as opposed to hyphen-case:

```js
var switchableClass = div.dataset.switchableClass
```